File: Vectors , ArrayList, List/C++/FindMaxElement.cpp
// Find the Maximum Element in Vector.
#include <iostream>
#include <vector>
using namespace std;

int main(){
    vector<int> v = {21,3,13,433,453,3};
    int maxElement = v[0];
    for (int val : v){
        if (val > maxElement){
            maxElement = val;
        }
    }
    cout << "Max Element: " << maxElement << endl;
}

File: Vectors , ArrayList, List/Java/FindMaxElement.java
// Find max element in a ArrayList
import java.util.ArrayList;
import java.util.Collections;

public class FindMaxElement {
    public static void main(String[] args) {
        ArrayList<Integer> v = new ArrayList<>();
        v.add(21);
        v.add(3);
        v.add(13);
        v.add(433);
        v.add(453);
        v.add(3);

        int maxElement = Collections.max(v);
        System.out.println("Max Element: " + maxElement);
    }
}